### DISCUSSION TITLE
1.0 or 1.0.0rc4

### DISCUSSION THREAD

**Discussion by alaingdl** [gnudatalanguage/gdl#953] [General]
Hi

Do you have a plan (date, deadline ?) for the 1.0.0 release ?

If not, is it OK to make now/quickly a 1.0.0rc4 delivery ?
 
(because 1.0.0rc3 is one year old, we made a lot of changes,
it is not easy to know which versions users are using ...
much more easy for me to answer : hey, it is working in 1.0.0rc4 !)

thanks

best regards

A.

**Comment by GillesDuvert** [comment]
There is an issue with the wxWidgets plot window when plotting an image.
Give me a couple of days, too busy elsewhere.

**Comment by pjb7687** [comment]
I am working on a Github Actions script to make the builds work again. I leave a note here about what I am currently working on:
1. The Ubuntu build works again, but it cannot finds HDF4 and GRIB.
2. On macOS CMake finds Python 2, while it correctly finds Numpy installed on Python 3. This seems to be the reason for the latest compile failure.
3. On Windows, CMake cannot find Numpy. I think I am going to finish #886 and merge it instead.

Another thing: I applied for early access to the GA image of macOS Big Sur. It would be nice we could ensure GDL works on Big Sur before the 1.0 release.

**Reply by pjb7687** [reply]
All done! https://github.com/gnudatalanguage/gdl/pull/971

**Reply by GillesDuvert** [reply]
yes, certainly having a BigSur test before release is Big Sure

**Reply by pjb7687** [reply]
I confirm GDL compiles well on Big Sur:
https://github.com/gnudatalanguage/gdl/runs/3231849752
